mockup/patterns/moment/pattern.js

```diff
@@ -86,7 +86,42 @@ define([
   'mockup-i18n',
   'moment'
 ], function($, Base, i18n, moment) {
-  lazyLoadMomentLocale();
+  var currentLanguage = (new i18n()).currentLanguage;
```

Comment:
Not a merge blocker, but is this needed here?
I would just evaluate it in the lazyLoadMomentLocale function.

Reply:
I moved this same line from a few lines below so I didn't have to repeat myself.

Also I preferred to fetch from mockup-i18n than directly from document.querySelector('html') so that if we change the implementation there, or if we want some transformations, Moment/js pattern get directly from there.

Reply:
Oh, I see that this is also used later in another method.